src/components/cards/OpRewardsCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { GenericCard } from '@/components/cards/GenericCard';
+import { Box, Cover, Stack, Text } from '@/design-system';
+import { ButtonPressAnimation } from '@/components/animations';
+import { Image, StyleSheet } from 'react-native';
+import OpRewardsCardBackgroundImage from '../../assets/opRewardsCardBackgroundImage.png';
+import { useTheme } from '@/theme';
+import * as i18n from '@/languages';
+import { useNavigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+
+const GRADIENT = ['#520907', '#B22824'];
+
+export const OpRewardsCard: React.FC = () => {
+  const { colors } = useTheme();
+  const { navigate } = useNavigation();
+
+  const navigateToRewardsSheet = () => {
+    navigate(Routes.OP_REWARDS_SHEET);
+  };
+
+  return (
+    <GenericCard
+      type="stretch"
+      gradient={GRADIENT}
+      onPress={navigateToRewardsSheet}
```

Comment:
do you want to add a shadow color? i haven't seen the designs but the other colored cards have correspondingly colored shadows. you could add `color={colors.networkColors.optimism}`

Reply:
Thanks, didn't know about it